shared-helpers/src/types/backend-swagger.ts

```diff
@@ -235,6 +235,28 @@ export class ListingsService {
       axios(configs, resolve, reject)
     })
   }
+  /**
+   * Get a paginated set of listings
+   */
+  filterableList(
```

Comment:
The swagger UI is working correctly, but the sample body is not valid parameters. For example, filter should be an array but it is just the first object in the sample.
```
{
  "page": 1,
  "limit": 10,
  "filter": {
    "$comparison": "=",
    "status": "active"
  },
  "view": "fundamentals",
  "orderBy": "mostRecentlyUpdated",
  "orderDir": "[\"desc\"]",
  "search": "search"
}
```
I wonder if there are things we could change in the `listings-query-params.dto.ts` file to make it work correctly

Reply:
Looks like the values are taken directly from `example` above each value. If those are updated it will fix the problem